Dockerfile

```diff
@@ -1,56 +1,26 @@
 FROM registry.fedoraproject.org/fedora:latest
 
-RUN dnf -y update && dnf install -y make git golang golang-github-cpuguy83-md2man \
-	# storage deps
-	btrfs-progs-devel \
-	device-mapper-devel \
-	# gpgme bindings deps
-	libassuan-devel gpgme-devel \
-	gnupg \
-	# htpasswd for system tests
-	httpd-tools \
-	# OpenShift deps
-	which tar wget hostname util-linux bsdtar socat ethtool device-mapper iptables tree findutils nmap-ncat e2fsprogs xfsprogs lsof docker iproute \
-        bats jq podman runc \
-	golint \
-	openssl \
-	&& dnf clean all
-
-# Install two versions of the registry. The first is an older version that
-# only supports schema1 manifests. The second is a newer version that supports
-# both. This allows integration-cli tests to cover push/pull with both schema1
-# and schema2 manifests.
-RUN set -x \
-	&& export GO111MODULE=off \
-	&& REGISTRY_COMMIT_SCHEMA1=ec87e9b6971d831f0eff752ddb54fb64693e51cd \
-	&& REGISTRY_COMMIT=47a064d4195a9b56133891bbb13620c3ac83a827 \
-	&& export GOPATH="$(mktemp -d)" \
-	&& git clone https://github.com/docker/distribution.git "$GOPATH/src/github.com/docker/distribution" \
-	&& (cd "$GOPATH/src/github.com/docker/distribution" && git checkout -q "$REGISTRY_COMMIT") \
-	&& GOPATH="$GOPATH/src/github.com/docker/distribution/Godeps/_workspace:$GOPATH" \
-		go build -o /usr/local/bin/registry-v2 github.com/docker/distribution/cmd/registry \
-	&& (cd "$GOPATH/src/github.com/docker/distribution" && git checkout -q "$REGISTRY_COMMIT_SCHEMA1") \
-	&& GOPATH="$GOPATH/src/github.com/docker/distribution/Godeps/_workspace:$GOPATH" \
-		go build -o /usr/local/bin/registry-v2-schema1 github.com/docker/distribution/cmd/registry \
-	&& rm -rf "$GOPATH"
+ENV GOPATH="/var/tmp/go"
+ENV GOSRC="$GOPATH/src/github.com/containers/skopeo"
+ENV PATH="$GOPATH/bin:$GOSRC/bin:/usr/share/gocode/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/bin:/sbin"
```

Comment:
(Is there a benefit to hard-coding the various default instead of just prepending to distribution-specified $PATH ?)

Reply:
My understanding is that you can't reference the container's `$PATH`  from inside an `ENV` instruction.  It will simply be blank (I need to confirm this).  Normally `$PATH` is setup by pam, and/or `/etc/profile*` + the `$HOME` config/script files.  So assuming there's even a shell executable inside the container, it would have to be executed to first obtain the value, then somehow pass that back into the build context.  So I think it _has_ to be hard-coded if we want to change it in the Dockerfile.  Checking...

Reply:
...okay, I stand corrected.  It looks like `podman` build either hard-codes in a default value, or is somehow obtaining it from inside the container.

Reply:
OK, if the previous version didn’t work, that’s a pretty strong reason.

Reply:
Nope, it probably did work.  I was wrong, so I put it back to the old way.